util/config.py

```diff
@@ -12,6 +12,9 @@ def get_default_version() -> str:
 
 def get_functions_csv_path(version = get_default_version()) -> Path:
     value = CONFIG["functions_csv"]
+    if version is None:
```

Comment:
Not a big fan of having `version` default to `get_default_version()` *and* having a check for None. Either get rid of this or change the default value for `version` to None.